internal/database/export.go

```diff
@@ -512,7 +519,28 @@ func (db *DB) DeleteFilesBefore(ctx context.Context, before time.Time, blobstore
 	return count, nil
 }
 
+// addExportFile adds a row to ExportFile. If the row already exists (based on the primary key),
+// ErrKeyConflict is returned.
 func addExportFile(ctx context.Context, tx pgx.Tx, ef *model.ExportFile) error {
+	row := tx.QueryRow(ctx, `
```

Comment:
Does the insert below return number of records inserted or anything? Maybe don't need a dedicated query here?

Reply:
Good idea. Let me try.

Reply:
Yup, works. Updated PR.